metrics_utils/src/lib.rs

```diff
@@ -177,9 +178,20 @@ impl BackfillerMetricsConfig {
             slots_collected: Family::<MetricLabelWithStatus, Counter>::default(),
             data_processed: Family::<MetricLabel, Counter>::default(),
             last_processed_slot: Family::<MetricLabel, Gauge>::default(),
+            slot_delay: Family::<MetricLabel, Histogram>::new_with_constructor(|| {
+                Histogram::new(exponential_buckets(400.0, 1.8, 10))
+            }),
         }
     }
 
+    pub fn set_slot_delay_time(&self, label: &str, duration: f64) {
+        self.slot_delay
+            .get_or_create(&MetricLabel {
+                name: label.to_string(),
```

Comment:
So, we should downgrade `String` to `str` and back? Probably, if we need a string then it's better to pass the `String` to avoid such behind-the-scenes allocations.

Reply:
Maybe `MetricLabel` can hold a `&str`?